editor.planx.uk/src/ui/shared/CharacterCounter.tsx

```diff
@@ -0,0 +1,73 @@
+import Typography from "@mui/material/Typography";
+import { visuallyHidden } from "@mui/utils";
+import { TEXT_LIMITS, TextInputType } from "@planx/components/TextInput/model";
+import { debounce } from "lodash";
+import React, { useCallback, useEffect, useState } from "react";
+import { FONT_WEIGHT_SEMI_BOLD } from "theme";
+
+export type Props = {
+  limit: TextInputType.Long | TextInputType.ExtraLong;
+  count: number;
+  error: boolean;
+};
+
+export const CharacterCounter: React.FC<Props> = ({ limit, count, error }) => {
+  const [screenReaderCount, setScreenReaderCount] = useState<number>(0);
+  const [showReaderCount, setShowReaderCount] = useState<boolean>(false);
+
+  const updateCharacterCount = useCallback(
+    debounce((count: number) => {
+      setScreenReaderCount(count);
+      setShowReaderCount(true);
+    }, 500),
+    []
+  );
+
+  const currentCharacterCount = TEXT_LIMITS[limit] - count;
+  const showCharacterLimitError = currentCharacterCount < 0;
+
+  useEffect(() => {
+    if (count !== screenReaderCount) {
+      setShowReaderCount(false);
+    }
+    if (count > 0) {
```

Comment:
This reduced the amount of noise coming when you enter the textarea, felt low key enough to sneak in...